draft/spec/validation-codes.md

```diff
@@ -115,6 +115,8 @@
 | E103 | 'Each version directory within an OCFL Object MUST conform to either the same or a later OCFL specification version as the preceding version directory.' | https://ocfl.io/draft/spec/#E103
 | E104 | 'Version directory names MUST be constructed by prepending v to the version number.' | https://ocfl.io/draft/spec/#E104
 | E105 | 'The version number MUST be taken from the sequence of positive, base-ten integers: 1, 2, 3, etc.' | https://ocfl.io/draft/spec/#E105
+| E106 | 'The value of the manifest key MUST be a JSON object.' | https://ocfl.io/draft/spec/#E106
+| E107 | 'The value of the manifest key must be a JSON object, and each key MUST correspond to digest value keys found in one or more state JSON objects of the current and/or previous version blocks of the OCFL Object.' | https://ocfl.io/draft/spec/#E107
```

Comment:
I don't want to be overly strict, but there are still minor differences in the text between the spec:
> The value of the manifest key MUST be a JSON object, and each key MUST correspond to a digest value key found in one or more state blocks of the current and/or previous version blocks of the OCFL Object.

..and the error code text:
> The value of the manifest key must be a JSON object, and each key MUST correspond to digest value keys found in one or more state JSON objects of the current and/or previous version blocks of the OCFL Object.

Reply:
Thanks again, fixed